source/ingress.go

```diff
@@ -70,6 +73,10 @@ func (sc *ingressSource) Endpoints() ([]*endpoint.Endpoint, error) {
 	if err != nil {
 		return nil, err
 	}
+	ingresses.Items, err = sc.filterByAnnotations(ingresses.Items, sc.annotationFilter)
```

Comment:
if you call a method of "sc", you should not pass sc.annotationFilter as parameter, but use it in the method from your receiver.